### PR TITLE
Separated supplemental resources- part 8

### PR DESCRIPTION
**Pull request by raibove**
Fixed issue: #836

### CODE REVIEW

File: command-line/command-line-advanced.md [hunk 1]
```diff
@@ -41,38 +45,6 @@ Teach people some handier tools for advanced file searching and processing
...
-- [Video](https://www.youtube.com/watch?v=nLa6jAbULe8): About 8 minutes. Covers command cut, &&, sort, uniq, wc, grep commands. 
-    - 0:59 && operator
-    - 3:34 cut command
-    - 4:28 sort command
-    - 4:59 uniq command
-    - 5:20 wc command
-    - 5:40 grep command
-
-- [Video](https://www.youtube.com/watch?v=yCTnihfbPCo): About 55 minutes. Covers command grep, sed, awk, tar, less, gzip commands.
-    - 1:25 grep command
```
Comment (raibove) [inline command-line/command-line-advanced.md]: I removed this timings . Is it ok?

File: command-line/dot-profile.md [hunk 2]
```diff
@@ -142,10 +135,6 @@ Nano is a Linux command line text editor. It is relatively easy to learn and use
 - Any command prefixed with the caret symbol(^) means to use the "ctrl" key. For example, **^G** means to press **ctrl+G** keys at he same time.
 - Any command prefixed with letter M means to press the Alt key. For example, **M-R** means to press **Alt+r** keys together at the same time.
 
-For more detailed info on how to use the nano tool, visit these sites below:
```
Comment (raibove) [inline command-line/dot-profile.md]: I moved them to supplemental resources as description to use this material is not provided.
Is it ok or shall I place them here again.

File: command-line/command-line-advanced.md (hunk 1 above)
Comment (alodahl) [reply command-line/command-line-advanced.md]: Actually we asked a participant to add those, please put them back =)

File: operating-systems/operating-systems.md [hunk 3]
```diff
@@ -99,3 +89,11 @@ Re-sort the processes in Activity Monitor to by CPU%. You will probably notice p
 ### Check for Understanding
 
 Pair with another participant who researched a different helper process. Explain to each other what you learned about the process, and discuss how you think affects the efficiency of the overall application. 
+
+### Materials
```
Comment (alodahl) [inline operating-systems/operating-systems.md]: Supplemental Materials?

File: command-line/dot-profile.md [hunk 4]
```diff
@@ -143,22 +153,9 @@ Add the following to your `.profile` change what your prompt displays: `export P
 Save & open a new Terminal window or `source ~/.profile` to reload this. Notice how the prompt now displays 
 something simliar to `david@Davids-MacBook-Pro` now.
 
-### Independent Practice
-[BASH Programming - Introduction HOW-TO](http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO.html)
-
+(Optional:) More info on - [How to customize your terminal prompt](http://osxdaily.com/2006/12/11/how-to-customize-your-terminal-prompt/)
 
-### Supplemental Material
+Go through the video tutorials mentioned in Material section.
```
Comment (alodahl) [inline command-line/dot-profile.md]: Please move this up to line 35 so it doesnt look optional.

I think that's it!  Thanks for fixing this up!

File: command-line/command-line-advanced.md [hunk 5]
```diff
@@ -3,10 +3,14 @@
...
 About 1 hour
+- 20 minutes for Lesson
+- 20 minutes for Guided Practice
+- 15 minutes for Independent Practice
+- 10 minutes for Check for Understanding
 
 ### Prerequisites
 
-[Basic command line class](command-line-interface.md).
+[Basic command line](command-line-interface.md).
```
Comment (Kundan28) [inline command-line/command-line-advanced.md]: (../command-line-interface.md)

Comment (raibove) [reply command-line/command-line-advanced.md]: If I change to this link then it says error 404 not found. @Kundan28 Please help to solve this issue

Comment (alodahl) [reply command-line/command-line-advanced.md]: (command-line-interface.md) is actually correct, since it is in the same folder as the outline.

**Comment by alodahl** [comment]
Be sure to solve your conflicts too, @raibove . Then we can merge!

**Comment by raibove** [comment]
There are no conflicts

**Comment by alodahl** [comment]
image

**Comment by Kundan28** [comment]
> There are no conflicts

IMG_20190526_225217

**Comment by alodahl** [comment]
The conflict notation needs to be deleted at least.

**Comment by raibove** [comment]
I resolved the conflicts. General Mac tips were added to file which wasn't updated in my branch these created the conflict

**Pull request merged**